actionview/lib/action_view/record_identifier.rb

```diff
@@ -80,17 +80,17 @@ def dom_class(record_or_class, prefix = nil)
     # If no id is found, prefix with "new_" instead.
     #
     #   dom_id(Post.find(45))       # => "post_45"
-    #   dom_id(Post.new)            # => "new_post"
+    #   dom_id(Post)                # => "new_post"
```

Comment:
The previous example still works so we should leave both examples in the docs.

Reply:
I left only the new syntax because that's what was documented for dom_class, and to encourage people to use the new one. It seems implicit to me that the old one still works, but I won't argue if you think it's best to leave both. Or should we add the `dom_class(Post.new)` example syntax too?